src/Translation/FileLoader.php

```
@@ -18,26 +18,31 @@ class FileLoader extends FileLoaderBase
      */
     protected function loadNamespaceOverrides(array $lines, $locale, $group, $namespace)
     {
-        $namespace = str_replace('.', '/', $namespace);
+        return collect($this->paths)
+            ->reduce(function ($output, $path) use ($lines, $locale, $group, $namespace) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$output'. (undefined)

(UnusedFormalParameter)